### DISCUSSION TITLE
Not able to publish site

### DISCUSSION THREAD

**Discussion by hasansaiyed** [community/community#28145] [New to GitHub] [answered]
Hello there,
I need help in publishing my site on github, it appears that it is showing READ.md file once i open my url, instead of the actual website created.

Once i am able to see the main page of the website, once i click on the link on the page, it takes me to another page shows error 404 not found.

**Comment by radar-blip** [comment, marked as answer]
Looking at your odin-repository, your links are pointing to a recipes folder, but you have no recipe folder in your repository. 
You need to create a recipe folder in order to get it working correctly.

And of course, you need to put your html files into the recipes folder (along with pictures).

Its kind of hard to explain so here is a short tutorial on creating a new folder on Github:
https://newbedev.com/how-do-i-create-a-folder-in-a-github-repository

Good Luck!

**Reply by hasansaiyed** [reply]
is there any way i can directly upload a whole file i.e. named recipes on my local directory, because i am not able to do it right now, as i am trying to upload files, that file folder is not getting selected, and i have to select the content within and upload them seprately

**Reply by hasansaiyed** [reply]
Also, as you said that i can create a file by using the method you suggested on github, but how can i add html and jpg files to it, because we cannot create an empty folder/file, can we?

**Reply by radar-blip** [reply]
Right, you cannot create an empty folder. You would have to open the individual files first and save them into a newly created Folder. It should show you how to do this in the link I sent you. 
Or, you could take the word 'recipes' out of your link and it should work that way also (without creating a folder).

**Reply by hasansaiyed** [reply]
Thank you

**Reply by radar-blip** [reply]
You're welcome! 👍🏼 

Another thing you can do is add a theme to your site. 
You can do this by opening your repository, go to:  Settings  >>  Pages  >>  Theme Chooser,  ...and pick one.
This will give your site a different appearence. You can always undo this if you dont like it.


![themes](https://user-images.githubusercontent.com/39196767/182939056-bbe618b5-81e8-462c-ade9-1779797cddb2.png)

**Comment by jfullstackdev** [comment]
it's already deployed and the default is `index.html`

I included it at the URL to be sure it's the one being loaded 

![image](https://user-images.githubusercontent.com/47092464/182573378-71f5655d-be14-4a6e-8102-c21b4054e1d7.png)

**Reply by hasansaiyed** [reply]
Yes it is deployed but once i click on this link then it shows error 404.

**Reply by jfullstackdev** [reply]
this is the 404, it's because of the URL 

<https://hasansaiyed.github.io/odin-repository/recipes/bbq-chicken-breast.html>

this is the correct one 

<https://hasansaiyed.github.io/odin-repository/bbq-chicken-breast.html>

![image](https://user-images.githubusercontent.com/47092464/182591079-02d6b415-3cea-42bb-a0e7-fd6a0d91f347.png)

just remove all `recipes` because there is no such folder in your repo

**Reply by hasansaiyed** [reply]
Thanks a lot

**Comment by queenofcorgis** [comment]
Thanks @jdevstatic and @linuxlawson for chiming in - @hasansaiyed don't forget to mark the answer in case others encounter similar issues ✅

**Reply by radar-blip** [reply]
You're welcome, I would give you a thumbs-up but I cant seem to select it. 

![blah](https://user-images.githubusercontent.com/39196767/182654623-d8f85be5-3594-426e-b8d1-200ea10f85fe.png)

**Reply by queenofcorgis** [reply]
Hi there - what browser are you in? Have you tried resizing the window?

**Reply by radar-blip** [reply]
Firefox, using Linux. Window is maximized. Dont worry about me though, it dont bother me none. 👍🏼

**Reply by jfullstackdev** [reply]
it's already reported and was already responded by one of the Staff members

**Comment by samuelpajunen** [comment]
Hey! I´'ve done too much and googled, and asked chatgpt, and asked yt and tired everything but still my site isnt publish. Why is that and how long ?
